src/main/java/club/issizler/okyanus/api/cmdnew/CommandOption.java

```diff
@@ -0,0 +1,21 @@
+package club.issizler.okyanus.api.cmdnew;
+
+/**
+ * Command Option converts to Command Dispacher
+ * @param <X> is value type of option
+ */
+public interface CommandOption<X> {
+
+    /**
+     * Gets option of value
+     * @return {@link OptionType}
+     */
+    OptionType type();
+
+    /**
+     * Gets value of option
+     * @return object
+     */
+    Object value();
```

Comment:
Assuming this returns `X`, can't you just make it return `X` instead of `Object`?

Reply:
yes i forgeted it